db/migrate/20180819185350_add_fields_to_relief_camp.rb

```diff
@@ -0,0 +1,12 @@
+class AddFieldsToReliefCamp < ActiveRecord::Migration[5.2]
+  def change
+    add_column :relief_camps, :latitude, :float
+    add_column :relief_camps, :longitude, :float
```

Comment:
longitude double precision,
    latitude double precision,

Reply:
fixing in my copy of this PR